Dockerfile

```diff
@@ -44,16 +44,16 @@ RUN cd /usr/local/src \
     && cmake --build . --parallel "$(nproc)" \
     && cmake --install . --prefix "/opt/zmap"
 
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
 LABEL org.opencontainers.image.source="https://github.com/zmap/zmap"
 
 RUN apt-get update \
     && apt-get install -y \
     libpcap0.8 \
-    libjson-c4 \
+    libjson-c5 \
     libjudydebian1 \
-    libhiredis0.14 \
+    libhiredis1.1.0 \
```

Comment:
I don't think we should need Redis anymore given that we removed this output module a while back.